src/app/shared/layout/header.component.ts

```diff
@@ -67,7 +69,7 @@ export class HeaderComponent implements OnInit {
       });
 
     this.auth.currentAuthState.subscribe((state) => {
```

Comment:
Since we don't check state, perhaps we can remove state here.
```suggestion
    this.auth.currentAuthState.subscribe(() => {
```